ptcg_cockatrice.xml

```diff
@@ -1357,8 +1355,7 @@ Whenever Exeggutor blocks a Pokémon with flying, it deals 3 damage to that Pok
 			<type>Pokémon - Fighting</type>
 			<pt>5/3</pt>
 			<tablerow>2</tablerow>
-			<text>First strike
-Whenever Hitmonlee attacks, you may have it deal damage equal to its power to target Pokémon. If you do, prevent all combat damage Hitmonlee would deal this turn.</text>
+			<text>Whenever Hitmonlee attacks, you may have it deal damage equal to its power to target Pokémon. If you do, prevent all combat damage Hitmonlee would deal this turn.</text>
```

Comment:
Did you make a change here? Seems identical.

Reply:
Removed First strike, figured it makes sense to have hitmonlee kick dudes for free instead of always striking first so you kind of have to guess who's gonna block him, makes the card more fun IMO